lib/ts/recipe/passwordless/types.ts

```diff
@@ -125,6 +129,9 @@ export type TypeInput = (
     // By default (=undefined) it is done in the Core
     getCustomUserInputCode?: (userContext: any) => Promise<string> | string;
 
+    emailDelivery?: EmailDeliveryTypeInput<TypeEmailDeliveryTypeInput>;
```

Comment:
- This object should be wherever we havr `createAndSendCustomEmail`? 
   - But in normalised input, we can have it all the time (up to yoo)
- We should mark the older `createAndSendCustomEmail` as deprecated.

Reply:
> This object should be wherever we have `createAndSendCustomEmail`? 
Yes
> We should mark the older createAndSendCustomEmail as deprecated.
okay. This is true for all the Recipes, right?

Reply:
> This is true for all the Recipes, right?

Yes.